RecoJets/JetAlgorithms/src/HEPTopTaggerV2.cc

```diff
@@ -0,0 +1,651 @@
+#include "../interface/HEPTopTaggerV2.h"
+
+// Do not change next line, it's needed by the sed-code that makes the tagger CMSSW-compatible.
+namespace external {
+
+//optimal_R fit
+double R_opt_calc_funct(double pt_filt) {
+  return 327./pt_filt;
+}
+
+bool HEPTopTaggerV2_fixed_R::_first_time = true;
+
+void HEPTopTaggerV2_fixed_R::print_banner() {
+  if (!_first_time) {return;}
+  _first_time = false;
+
+  std::cout << "#--------------------------------------------------------------------------\n";
+  std::cout << "#                   HEPTopTaggerV2 - under construction                      \n";
+  std::cout << "#                                                                          \n";
+  std::cout << "# Please cite JHEP 1010 (2010) 078 [arXiv:1006.2833 [hep-ph]]              \n";
+  std::cout << "# and Phys.Rev. D89 (2014) 074047 [arXiv:1312.1504 [hep-ph]]               \n";
+  std::cout << "#--------------------------------------------------------------------------\n";
```

Comment:
cout is not allowed in production code.
In multi-threaded execution this text will likely be garbled.